Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -1528,7 +1528,7 @@ public RelativeDailyVolume RDV(Symbol symbol, int period = 2, Resolution resolut
 
             if (EnableAutomaticIndicatorWarmUp)
             {
-                WarmUpIndicator(symbol, relativeDailyVolume, resolution);
+                WarmUpIndicator(symbol, relativeDailyVolume, Resolution.Daily); // Fixed WarmUp Resolution because this indicator is based on days
```

Comment:
Cool!
Please add a test reproducing the issue and being fixed because of this change.
The default resolution of the indicators input data, line 1523 Daily, should probably be hourly? Else I don't think it would really be useful/valid? Seems like `RDV` should probably throw an argument exception if resolution is Daily